crates/astria-composer/src/executor/tests.rs

```diff
@@ -395,8 +468,18 @@ async fn two_seq_actions_single_bundle() {
     .build()
     .unwrap();
 
+    let nonce_guard = mount_nonce_query_mock(
+        &sequencer,
+        "accounts/nonce",
+        NonceResponse {
+            height: 0,
+            nonce: 0,
+        },
+    )
+    .await;
```

Comment:
nit: this is repeated a number of times perhaps a helper that does exactly this `mount_default_nonce_query_guard` function?

Reply:
Done in be387af.